src/main/resources/entity-types/finance/simple_groups.json5

```diff
@@ -0,0 +1,229 @@
+{
+  id: 'a01d982c-2c59-408e-8771-7ce0a07bc1a3',
+  name: 'simple_groups',
```

Comment:
nit: i'd prefer if it was `simple_finance_groups`, to disambiguate from user's patron groups